lib/dynamodb/set.js

```diff
@@ -25,6 +25,11 @@ var DynamoDBSet = util.inherit({
   initialize: function(list, validate) {
     var self = this;
     self.values = [].concat(list);
+    if (typeof Symbol !== 'undefined' && Symbol.iterator) {
```

Comment:
Please add a test in [test/dynamodb/converter.spec.js](https://github.com/aws/aws-sdk-js/blob/master/test/dynamodb/converter.spec.js)

Reply:
Done!